src/main/java/net/imglib2/algorithm/metrics/segmentation/SegmentationHelper.java

```diff
@@ -0,0 +1,68 @@
+package net.imglib2.algorithm.metrics.segmentation;
+
+import net.imglib2.roi.labeling.ImgLabeling;
+import net.imglib2.type.numeric.IntegerType;
+import net.imglib2.view.Views;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Helper class used to check if an {@link ImgLabeling} has intersecting labels.
+ *
+ * @author Joran Deschamps
+ */
+public class SegmentationHelper
+{
+
+	/**
+	 * Return a {@link Set} of occurring pixel values in the {@link ImgLabeling} index image.
+	 *
+	 * @param img
+	 * 		Image labeling from which to extract the occurring pixel values
+	 * @param <T>
+	 * 		Label type associated to the ground-truth
+	 * @param <I>
+	 * 		Image pixel type
+	 *
+	 * @return {@link Set} of occurring pixel values
+	 */
+	static < T, I extends IntegerType< I > > Set< I > getOccurringLabelSets( ImgLabeling< T, I > img )
```

Comment:
Isn't this something that [`LabelRegions#getExistingLabels()`](https://github.com/imglib/imglib2-roi/blob/f1e5daae75cab827790293b0bd9c9859f160d7c8/src/main/java/net/imglib2/roi/labeling/LabelRegions.java#L132-L136) already provides?

Reply:
See https://github.com/imglib/imglib2-roi/pull/59